hypervisor/dm/vuart.c

```diff
@@ -376,8 +390,31 @@ static void write_reg(struct acrn_vuart *vu, uint16_t reg, uint8_t value_u8)
 	release_vuart_lock(vu, rflags);
 }
 
-/*
- * @pre: vu != NULL
+/**
+ * @brief Write a value to a register in the virtual UART.
+ *
+ * This function writes a 8-bit value to a specified register in the virtual UART (vuart). It is a basic function used
+ * for port I/O write or MMIO write.
+ *
+ * - When the following conditions are met, data is sent to the target vUART's RXFIFO:
+ *   1) The target vUART exists for the specified vUART, which is used for communication.
+ *   2) The accessed register is the THR (Transmitter Holding Register).
+ *   3) Loopback mode is not enabled.
+ *   4) DLAB (Divisor Latch Access Bit) is not set.
+ * - Additionally, to ensure reliable communication, it raises the THRE interrupt (indicating that more data can be
```

Comment:
Make `- Additionally` aligned with `4) DLAB`? Because this logic is only effective when the conditions we described above are met.